src/main/telemetry/smartport.c

```diff
@@ -452,7 +448,7 @@ void handleSmartPortTelemetry(void)
 #ifdef GPS
             case FSSP_DATAID_GPS_ALT    :
                 if (sensors(SENSOR_GPS) && STATE(GPS_FIX)) {
-                    smartPortSendPackage(id, GPS_altitude * 1000); // given in 0.1m , requested in 100 = 1m
+                    smartPortSendPackage(id, GPS_altitude * 100); // given in 0.1m , requested in 10 = 1m
```

Comment:
probably bug in openTx - when converting units, precision is only decreased, so no conversion is done. Just comment that sent value should be in mm ...